scripts/artifacts-building/apt/apt-artifacts-testing.yml

```diff
@@ -0,0 +1,60 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: no
+  vars:
+    distro: "trusty"
```

Comment:
Why is this not hard coded like it is in the nested loops below. Either this needs to be something that can be changed when testing, or it should be hard coded to test everything, always.